Makefile

```diff
@@ -0,0 +1,10 @@
+CC = gcc
+CFLAGS = -g -Wall
```

Comment:
`CFLAGS` is a special variable name and I set it so that you had more compiler flags for stricter build. You get more warnigns and errors essentially. You will always use this for AP.